build.sbt

```diff
@@ -9,19 +9,22 @@ lazy val gcsFileSystem = (project in file("filesystems/gcs"))
   .settings(gcsFileSystemSettings:_*)
   .withTestSettings
 
-lazy val database = (project in file("database"))
-  .settings(databaseSettings:_*)
-  .dependsOn(core) // used for migration
-  .dependsOn(core % "test->test") // TODO: PBE: Remove once PostMVP tag removed from test
+lazy val databaseSql = (project in file("database/sql"))
+  .settings(databaseSqlSettings:_*)
+  .withTestSettings
+
+lazy val databaseMigration = (project in file("database/migration"))
+  .settings(databaseMigrationSettings: _*)
+  .dependsOn(core)
   .withTestSettings
 
 lazy val services = (project in file("services"))
   .settings(servicesSettings:_*)
   .withTestSettings
   .dependsOn(core)
-  .dependsOn(database) // Assuming services impl in services
+  .dependsOn(databaseSql)
+  .dependsOn(databaseMigration)
```

Comment:
Yep, we apparently had created a full services layer under `cromwell-services`. I quite fond of the way the separation of concerns has emerged, in that `cromwell-backend`/`cromwell-engine` talks to the `cromwell-services` that talks to the `cromwell-database` (if necessary).